Britannia/localization/english/BTWK_storyline_events_l_english.yml

```diff
@@ -81,17 +106,17 @@
  BTWK_story_event.0110.desc.header:0 "Uther Pendragon, the King of Dumnonia, is dead. The only legitimate heir to the throne is his clubfooted grandson Mordred, a mere infant.\n\n" 
  BTWK_story_event.0110.desc.was_attending_glevum:0 "With Uther's passing, the High Council can no longer go ahead. The grand assembly, designed to ensure unity remains among the Britons once the throne of the High King lay empty, will never come to pass.\n\n"
  BTWK_story_event.0110.desc.footer:0 "While Britannia's future is now as uncertain as ever, there exists still a glimmer of hope for the Britons. Upon receiving the news of his father's death, Uther's bastard son - and famed warlord in his own right - Arthur has returned from the Kingdom of Benoic, being appointed Lord of Durnac by the current regent. "
- BTWK_story_event.0110.a:0 "Hail to King Mordred and Lord Arthur!"
+ BTWK_story_event.0110.a:0 "Hail to King Mordred, and to Arthur!"
  BTWK_story_event.0110.b:0 "Let us hope Arthur drives the Saxons from Britain once and for all!"
  BTWK_story_event.0110.c:0 "They are weak. We will conquer their lands, no matter their leader."
  BTWK_story_event.0110.d:0 "This is interesting news."
 
- BTWK_story_event.0111.t:0 "From One Pendragon to Another"
- BTWK_story_event.0111.desc:0 "Your grandfather, King Uther Pendragon, has died. Prince Arthur, his bastard son and your uncle, has returned to Dumnonia from the Kingdom of Benoic upon hearing of Uther's death, where he had been sent to assist King Ban in his wars against the Franks. Upon his return, your current regent appointed him the Prince of Durnac. He has been hailed, like his father, as #italic Pendragon #. "
+ BTWK_story_event.0111.t:0 "Death of King Uther"
+ BTWK_story_event.0111.desc:0 "Your grandfather, King Uther Pendragon, has died. Arthur, his bastard son and your uncle, has returned to Dumnonia from his exile in Benoic to fulfil the oath made to Uther - ensuring you and your throne are protected. Upon returning, your council has arranged his appointment to the lands of Durnac, his role to manage your estates and armies till you are of fit age to rule them."
```

Comment:
1. "ensuring you and your throne are protected" -> "to protect you and your throne"
2. "Upon returning, your council has arranged his appointment to the lands of Durnac, his role to manage your estates and armies till you are of fit age to rule them." -> "Upon his return, he was given control over Durnac to support him and his warband during his guardianship/regency" ?

Reply:
1. Amended.
2. Not sure. "Upon his return, your council granted him control over Durnac for the duration of his guardianship, to support he and his warband. in doing so, his ability to protect you and your realm from harm's way has been greatly bolstered."?

Reply:
2. Hmm. I think it's probably better to avoid mentioning who specifically granted it to Arthur (it's not that important and also uh there isn't actually a good answer, it just always happens for gameplay purposes). The second sentence is okay but feels a bit redundant (it's implied in "supporting him and his warband").

Reply:
Okay then, "Upon his return, he was given control over Durnac for the duration of his guardianship to support he and his warband"?

Reply:
Maybe omit "for the duration of his guardianship" since it implies that Arthur is going to give it back after Mordred comes of age, which I'm not sure about...